Explorer/Assets/DCL/InWorldCamera/InWorldCamera/UI/InWorldCameraController.cs

```diff
@@ -0,0 +1,155 @@
+﻿using Arch.Core;
+using Cysharp.Threading.Tasks;
+using DCL.Audio;
+using DCL.CharacterCamera;
+using DCL.ExplorePanel;
+using DCL.InWorldCamera.CameraReelStorageService;
+using DCL.InWorldCamera.CameraReelStorageService.Schemas;
+using DCL.InWorldCamera.Playground;
+using DCL.UI;
+using ECS.Abstract;
+using JetBrains.Annotations;
+using MVC;
+using System.Diagnostics;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.InWorldCamera.UI
+{
+    /// <summary>
+    /// Handles Logic for the InWorldCamera HUD that appears when user enables InWorldCamera.
+    /// </summary>
+    public class InWorldCameraController : ControllerBase<InWorldCameraView>
+    {
+        private readonly Button sidebarButton;
+        private readonly World world;
+        private readonly IMVCManager mvcManager;
+        private readonly ICameraReelStorageService storageService;
+
+        private ScreencaptureShortcutsController shortcutsController;
+        private SingleInstanceEntity? cameraInternal;
+
+        private bool shortcutPanelIsOpen;
+
+        public override CanvasOrdering.SortingLayer Layer => CanvasOrdering.SortingLayer.Overlay;
```

Comment:
being an overlay UI, what happens (if it's possible) if from this overlay the loading screen appears?
Asking because I think the MVCManager and windows stack manager currently don't support a stack of overlay UIs but only 1 at a time 🤔